### ISSUE TITLE
Enhancement: Responsive design

### ISSUE THREAD

**Issue by hbayyavarapu** [dabeng/OrgChart#4]
This plugin does a perfect job for the org chart. However, it would be really awesome if its responsive too.

**Comment by dabeng** [comment]
You mean it's better to add responsive feature to orgchart plugin so that it could work properly on smartphones, tablets as well as desktop devices.

**Comment by hbayyavarapu** [comment]
yes, exactly. Its a top requirement for my organization, everything has to
be responsive.

**Comment by Djalmar** [comment]
+1

**Comment by DelightedD0D** [comment]
+1 

I like the _idea_ of the chart being responsive down to say tablets, but i'm not sure it's really practical to display data like this down to the iPhone level. In my opinion, it might be better if phones hid the chart altogether and showed a li menu structure in it's place with simple dropdown controls for moving the elements instead of using drag and drop with tiny elements.

**Comment by similiarforce** [comment]
I have been playing with this library with large amount of nodes [around 500 i think but it could go to thousands]...

I am using latest chrome and I found it bit difficult to work with zoom and pan.

By difficult I mean it takes a lot of effort to move from left side nodes to right side or other way around... if we have lots of them in parallel.

I will try to recreate the issue at some point and demonstrate but I think it's related to CPU usage as when I move this huge chart using pan, I can see CPU keep hitting 100%...

Maybe that's why it's not moving lightly or freely if I am using correct words.

Many thanks

**Comment by DelightedD0D** [comment]
@similiarforce you should start a new issue for your problem

**Comment by similiarforce** [comment]
@DelightedD0D I thought it was related so added it here, but I have moved it now, thanks :)

**Comment by jonatasu** [comment]
+1

**Comment by gangesh** [comment]
+1 
Is this thread even followed by authors of this plugin?

**Comment by ManuDevelopia** [comment]
+1 Any news? would be very nice to have this support

**Comment by dabeng** [comment]
Thank you guys for your great idea. It will be a great feature. I'll take the time to implement in the version 2.1.4 😁

**Comment by khaivngo** [comment]
> Thank you guys for your great idea. It will be a great feature. I'll take the time to implement in the version 2.1.4 😁

Great! Any ETA on next release?

**Comment by dabeng** [comment]
Hi all. I find that responsive design feature based on pure css doesn't work. So I make use of existing init() method to implement it. Is this demo(https://dabeng.github.io/OrgChart/responsive-design.html) what you want ? 😊
